docs/content/latest/explore/_index.md

```diff
@@ -136,52 +148,30 @@ Use [yb-ctl](../admin/yb-ctl) local cluster management utility to explore the co
 -->
 
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="observability/">
+    <a class="section-link icon-offset" href="security/security/">
       <div class="head">
-        <img class="icon" src="/images/section_icons/explore/monitoring.png" aria-hidden="true" />
-        <div class="title">Observabililty</div>
+        <img class="icon" src="/images/section_icons/explore/secure.png" aria-hidden="true" />
+        <div class="title">Security</div>
       </div>
       <div class="body">
-        Observe metrics at every layer of the database using Prometheus.
+        Learn about strategies for securing YugabyteDB.
       </div>
     </a>
   </div>
 
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="extensions/">
+    <a class="section-link icon-offset" href="observability/">
       <div class="head">
         <img class="icon" src="/images/section_icons/explore/monitoring.png" aria-hidden="true" />
-        <div class="title">Extensions</div>
+        <div class="title">Observabililty</div>
       </div>
       <div class="body">
-        Take advantage of powerful pre-bundled extensions.
+        Observe metrics at every layer of the database using Prometheus.
       </div>
     </a>
   </div>
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="linear-scalability/sharding-data/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
-        <div class="title">Sharding strategies</div>
-      </div>
-      <div class="body">
-        Learn about range and hash sharding strategies.
-      </div>
-    </a>
-  </div>
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="security/security/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/secure.png" aria-hidden="true" />
-        <div class="title">Security</div>
-      </div>
-      <div class="body">
-        Learn about strategies for securing YugabyteDB.
-      </div>
-    </a>
-  </div>
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
     <a class="section-link icon-offset" href="cluster-management/point-in-time-recovery-ysql/">
       <div class="head">
```

Comment:
Minor - Cluster Management tile points to PITR instead of Cluster Mgmt; given there is only 1 topic that's prob fine; but it is also pointing to YSQL; elsewhere YCQL is preferred as it is not BETA

Reply:
Fixed